tests/unit/test-util.js

```diff
@@ -22,6 +29,14 @@ const createConnStub = (testCase) => {
   };
 };
 
+/**
+ * @summary Transoform the rawData into serializedData.
```

Comment:
You can remove the @summary tag and the `@function` line. [Example](https://github.com/osu-mist/game-curator-api/blob/4929a92be94da94b9fb2084382360c9832da033b/tests/unit/test-helpers.js#L38-L44).